src/main/java/uk/gov/companieshouse/api/testdata/model/entity/CompanyProfile.java

```diff
@@ -5,26 +5,67 @@
 import org.springframework.data.mongodb.core.mapping.Document;
 import org.springframework.data.mongodb.core.mapping.Field;
 
+import java.time.Instant;
+import java.util.List;
+
 @Document(collection = "company_profile")
 public class CompanyProfile {
 
     @Id
     @Field("id")
     private String id;
-    @Field("company_name")
-    private String companyName;
+    @Field("links")
+    private Links links;
+    @Field("accounts.next_due")
+    private Instant accountsNextDue;
+    @Field("accounts.next_accounts.period_start_on")
+    private Instant periodStart;
+    @Field("accounts.next_accounts.period_end_on")
+    private Instant periodEnd;
+    @Field("accounts.next_accounts.due_on")
+    private Instant nextAccountsDueOn;
+    @Field("accounts.next_accounts.overdue")
+    private Boolean nextAccountsOverdue;
+    @Field("accounts.next_made_up_to")
+    private Instant accountsNextMadeUpTo;
+    @Field("accounts.accounting_reference_date.day")
+    private String accountingReferenceDateDay;
+    @Field("accounts.accounting_reference_date.month")
+    private String accountingReferenceDateMonth;
```

Comment:
Would it be worth grouping the account fields in its own entity for the sake of readability/maintainability? 
Logically it makes sense to have an object for them and we could do it in a similar way the `CompanyMetrics` does it for pscs and officers. And possibly do it for the confirmation statement too?